frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ORDERS, SAMPLE_DATABASE } from "__support__/sample_database_fixture";
+import StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import ExpressionWidget, { ExpressionWidgetProps } from "./ExpressionWidget";
```

Comment:
I think lately we've been adopting `import type` for types. I think you should extract the type import to another line what do you think @kulyk I think you also use `import type` a lot too.